### ISSUE TITLE
Dependency number is undefined for relative includes when specifying many files

### ISSUE THREAD

**Issue by ptarjan** [substack/node-browserify#1266]
With this simple setup

```
$ cat a.js
var b = require('./b.js');
console.log(b.foo());
$ cat b.js
exports.foo = function() {
  return 'works';
};
```

Then with version 9.0.8 I happily get `"./b.js":2`

```
$ browserify --version
9.0.8
15:16:16 pt@Pauls-MBP:/tmp/browsesrify$ browserify a.js b.js
(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
var b = require('./b.js');
console.log(b.foo());

},{"./b.js":2}],2:[function(require,module,exports){
exports.foo = function() {
  return 'works';
};

},{}]},{},[1,2]);
```

But with 10.2.0

```
$ node_modules/browserify/bin/cmd.js --version
10.2.0
$ node_modules/browserify/bin/cmd.js a.js b.js
(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
var b = require('./b.js');
console.log(b.foo());

},{"./b.js":undefined}],2:[function(require,module,exports){
exports.foo = function() {
  return 'works';
};

},{}]},{},[1,2]);
```

Note for just a single file it works:

```
$ node_modules/browserify/bin/cmd.js a.js
(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
exports.foo = function() {
  return 'works';
};

},{}],2:[function(require,module,exports){
var b = require('./b.js');
console.log(b.foo());

},{"./b.js":1}]},{},[2]);
```

**Comment by zertosh** [comment]
Sorry @ptarjan, I broke that in 10.1.1. Prior to that, your example would've worked via the CLI but not API - we're trying to sort those differences out. But https://github.com/substack/node-browserify/pull/1268 fixes it, I'll get that merged soon.

**Comment by ptarjan** [comment]
Thanks for the quick fix.

**Comment by ptarjan** [comment]
I don't quite understand your process. Is this fixed in 10.2.3?

**Comment by zertosh** [comment]
@ptarjan sorry this is being held up. I'll merge it tmw.

**Comment by zertosh** [comment]
the fix is in 10.2.4
